Glyssen/Bundle/GlyssenDblTextMetadata.cs

```diff
@@ -263,6 +268,27 @@ public QuoteSystemStatus QuoteSystemStatus
 		public VoiceActorStatus VoiceActorStatus { get; set; }
 	}
 
+	public class ProjectSettings
+	{
+		public const int kNumberOfNarratorsNotSet = 0;
+
+		/// <summary>
+		/// Number of narrators set by user
+		/// 0 if not set by user
+		/// </summary>
+		/// 
+		[XmlElement("numberofnarrators")]
```

Comment:
Typically, we use attributes for simple data types and elements for more complex stuff. Did you consider using attributes? If so, why did you choose to use elements? Because there might end up being a lot of settings, I could maybe see the benefit in using elements (to make the XML easier to read). Another option would be to have sub-elements under ProjectSettings to group the different settings. For now, we would just have "NarratorSettings", but eventually there would be others. Then each of those elements could use attributes for the simple int/bool/enum types.